### ISSUE TITLE
non english language

### ISSUE THREAD

**Issue by ifgcguitarclub** [rishikksh20/rishikksh20#3]
is it possible apply on other languages ( non english based speech) ?
thank you

**Comment by rishikksh20** [comment]
@ifgcguitarclub yes, TTS are sepuence to sequence you can convert one lang. seq to Mel spectrogram freq seq

**Comment by abhishek9466** [comment]
how to do it for hindi language

**Comment by rishikksh20** [comment]
Just replace the roman alphabet symbols with Devnagri symbols
